src/stactools/glad_glclu2020/commands.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from typing import Optional
+
+import click
+from click import Command, Group
+from pystac import MediaType
+
+from stactools.glad_glclu2020 import stac
+from stactools.glad_glclu2020.metadata import (
+    DEFAULT_HREF_FORMAT,
+    CollectionIDs,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def create_gladglclu2020_command(cli: Group) -> Command:
+    """Creates the stactools-glad-glclu2020 command line utility."""
+
+    @cli.group(
+        "gladglclu2020",
+        help="Generate STAC metadata for the GLAD Global Land Use Land Cover dataset",
+        short_help=("Commands for working with stactools-glad-glclu2020"),
+    )
+    def gladglclu2020() -> None:
+        pass
+
+    @gladglclu2020.command(
+        "create-collection",
+        help=stac.CREATE_COLLECTION_DESCRIPTION,
+        short_help="Creates a STAC collection",
+    )
+    @click.argument("destination")
+    @click.option(
+        "--type",
+        "collection_type",
+        type=click.Choice(["annual", "change"], case_sensitive=False),
+        required=True,
+        help="Type of collection to create",
+    )
+    @click.option(
+        "--media-type",
+        type=str,
+        help="Media type for the collection",
+        default=None,
```

Comment:
Should this default to the source GeoTiff?

Reply:
Yeah I think that would be a reasonable fallback since the default behavior is to make items for the original non-COGs.